spec/integration/elasticsearch/transform/manifest.spec.yml

```diff
@@ -1,18 +1,45 @@
-##
-## Describes the specification for a transform's manifest.yml file
-##
-spec:
```

Comment:
We have removed the `spec` level from this file, so we are effectively removing validation from here. Attempting to fix this in https://github.com/elastic/package-spec/pull/964/.